### ISSUE TITLE
Missing From Address does not throw any exceptions

### ISSUE THREAD

**Issue by zyofeng** [feinoujc/Mandrill.net#161]
Im not sure if this is a Mandrill Api issue or the lib, seeing this is just an httpclient wrapper I think the fault might fall on Mandrill Api

So the problem is if I SendAsync an email without a FromEmail address no errors are generated but the email basically disappears without a trace (does not actually get sent)

Any ideas on how to solve this?

**Comment by zyofeng** [comment]
Yes it's returning status of rejected with 200, should this throw an exception? (Maybe RejectedException and InvalidException)?

![image](https://user-images.githubusercontent.com/23351416/166835269-21dc6a10-f0b8-4f0e-9e33-54caf51dbe0d.png)

**Comment by feinoujc** [comment]
I think the problem--and the reason they designed their api this way, returning 200--is that its returning an array of results (you can send to multiple recipients) and its not obvious what to do for partial failure in that case.

What would you expect here if one recipient was sent and one was rejected?

**Comment by zyofeng** [comment]
good point since the response is an array. I guess there isn't much to be done on the lib side
